Add note about inertia being required for apply_torque on various Node types

compmstr opened the pull request:
see: https://github.com/godotengine/godot/issues/68068

*Bugsquad edit:*
- Fixes #68068

Calinou commented:
The same note is *likely* relevant for RigidBody3D and PhysicsDirectBodyState3D. Can you check there?

compmstr commented:
Checked in the code, and calculating angular velocity uses _inv_inertia_tensor, so it appears that inertia is required for 3D as well.  Added a similar note to apply_torque functions for RigidBody3D

mhilbrunner commented:
Please squash the commits into one :)

compmstr commented:
I think that commit squashing is done upon merging?  Or am I missing an option in Github?

Zireael07 commented:
@compmstr: It can be done upon merging, but Godot engine repo doesn't use this option

compmstr commented:
I followed https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html#the-interactive-rebase, and I think I've got it set up correctly.  TIL about manually squashing commits, thank you :)

I also added the note to PhysicsDirectBodyState3D.

compmstr commented:
I fixed the CI issues:
* had to change 'param' to 'member' for the inertia members
* I had the class reference tags wrong (it's just `[ClassName]` instead of `[class ClassName]`)

akien-mga commented on doc/classes/PhysicsDirectBodyState2D.xml:
```diff
@@ -76,6 +76,7 @@
 			<param index="0" name="torque" type="float" />
 			<description>
 				Applies a rotational force without affecting position. A force is time dependent and meant to be applied every physics update.
+				[b]Note:[/b] [member inverse_inertia] is required for this to work.  To have [member inverse_inertia], an active [CollisionShape2D] must be a child of the node, or you can manually set [member inverse_inertia].
```
We only use one space after a period in the docs. (Same for all added sentences.)

akien-mga commented:
Note that the email associated with your Git identity for this commit is not valid (see https://github.com/godotengine/godot/pull/68098.patch). It's not a problem per se but it means the commit can't be attributed to your GitHub account. If you want to fix that, you should change your Git identity locally and amend the commit with `git commit --amend --reset-author`.
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address#setting-your-commit-email-address-in-git

compmstr committed:
Add note about inertia being required for apply_torque on various Nodes

  - RigidBody2D
  - PhysicsDirectBodyState2D
  - RigidBody3D
  - PhysicsDirectBodyState3D

compmstr commented:
Thanks for the note about the email address.  I fixed that, and the double spaces after periods.

akien-mga commented:
Thanks! And congrats for your first merged Godot contribution :tada:
